src/main/java/tds/exam/results/mappers/OpportunityMapper.java

```diff
@@ -37,6 +36,9 @@
     private static final String DEFAULT_ALGORITHM_VERSION = "0";
     private static final String EXAM_DATABASE_NAME = "exam";
 
+    //This is the hard coded value for score dimensions in the TDS TRT based on ReportingDLL
+    private static final String SCORE_DIMENSION_VALUE = "overall";
```

Comment:
This is the change.  After looking at [this code](https://github.com/SmarterApp/TDS_TestDeliverySystemDataAccess/blob/master/tds-dll-mysql/src/main/java/tds/dll/mysql/ReportingDLL.java#L1068) in ReportingDLL, which creates the TRT, I realized that they don't use the score dimension as stored on the response but instead hard code it.

Just carrying that forward.